src/__tests__/AsyncCreatable.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+
+import AsyncCreatable from '../AsyncCreatable';
+import Select from '../Select';
+import { components } from '../components';
+const { Option } = components;
+
+test('defaults - snapshot', () => {
+  const tree = mount(<AsyncCreatable />);
+  expect(toJson(tree)).toMatchSnapshot();
+});
+
+test('creates an inner Select', () => {
+  const asyncCreatableWrapper = mount(<AsyncCreatable />);
+  expect(asyncCreatableWrapper.find(Select).exists()).toBeTruthy();
+});
+
+test('render decorated select with props passed', () => {
+  const asyncCreatableWrapper = mount(<AsyncCreatable className="foo"/>);
+  expect(asyncCreatableWrapper.find(Select).props().className).toBe('foo');
+});
+
+test('to show the create option in menu', () => {
```

Comment:
Please adjust this case to account for the fact that we only want to show the create option element, after the options have been loaded asynchronously.

Reply:
done